### PR TITLE
Antifraud Provider Update Parameters & Headers

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to update tje Antifraud Provider Protocol API:
1) Removal of unnecessary headers in documentation (https://vtex.slack.com/archives/C0165UQ8SSE/p1699988587589999).
2) Return of incorrectly removed parameters in PR-1037.

### CODE REVIEW

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Antifraud Provider API.json [hunk 1]
```diff
@@ -1432,6 +790,28 @@
...
+            "X-PROVIDER-API-AppToken": {
+                "name": "X-PROVIDER-API-AppToken",
+                "in": "header",
+                "description": "The AppToken configured by the merchant.",
+                "required": true,
+                "style": "simple",
+                "schema": {
+                    "type": "string",
+                    "example": "{{X-PROVIDER-API-AppToken}}"
+                }
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Antifraud Provider API.json]: Boa. Revisado.

File: VTEX - Antifraud Provider API.json [hunk 2]
```diff
@@ -751,15 +751,13 @@
                 "in": "header",
                 "name": "X-PROVIDER-API-AppKey",
                 "description": "The AppKey configured by the merchant."
```
Comment (karenkrieger) [inline VTEX - Antifraud Provider API.json:753]:
```suggestion
                "description": "Unique identifier of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys)."
```

File: VTEX - Antifraud Provider API.json [hunk 3]
```diff
@@ -751,15 +751,13 @@
...
-                }
             },
             "X-PROVIDER-API-AppToken": {
                 "type": "apiKey",
                 "in": "header",
                 "name": "X-PROVIDER-API-AppToken",
                 "description": "The AppToken configured by the merchant."
```
Comment (karenkrieger) [inline VTEX - Antifraud Provider API.json:759]:
```suggestion
                "description": "Secret token of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys)."
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**